OSDOCS-16940: DITA abstracts — OLM admin/user topics (#115359)
openshift/openshift-docs: modules/olm-preparing-upgrade.adoc

[id="olm-preparing-upgrade_{context}"]
= Preparing for an Operator update

> [role="_abstract"]
> An installed Operator subscription in {product-title} tracks updates through an update channel that you can change to receive newer versions. Update channel names typically follow the Operator's release stream or update frequency.

Reviewer: ```suggestion
[role="_abstract"]
The subscription of an installed Operator specifies an update channel that tracks and receives updates for the Operator. You can change the update channel to start tracking and receiving updates from a newer channel.
```
